PixivModelFanbox.py

```diff
@@ -193,12 +179,22 @@ def parseBody(self, jsPost):
                                      self.body_text,
                                      jsPost["body"]["imageMap"][imageId]["originalUrl"],
                                      jsPost["body"]["imageMap"][imageId]["thumbnailUrl"])
+                    for image in jsPost["body"]["imageMap"]:
+                        if image == imageId:
+                            self.images.append(jsPost["body"]["imageMap"][image]["originalUrl"])
+                            self.embeddedFiles.append(jsPost["body"]["imageMap"][image]["originalUrl"])
+                            break
                 elif block["type"] == "file":
                     fileId = block["fileId"]
                     self.body_text = u"{0}<br /><a href='{1}'>{2}</a>".format(
                                      self.body_text,
                                      jsPost["body"]["fileMap"][fileId]["url"],
                                      jsPost["body"]["fileMap"][fileId]["name"])
+                    for filename in jsPost["body"]["fileMap"]:
```

Comment:
same with this one, you can append the list directly without iterating.

```suggestion
                    self.images.append(jsPost["body"]["fileMap"][fileId]["url"])
                    self.embeddedFiles.append(jsPost["body"]["fileMap"][fileId]["url"])
```